test/test_data/set.py

```diff
@@ -0,0 +1 @@
+{1, 2, 3}
```

Comment:
Suggestion: I would expressions (like `1+2`) in these examples as they can catch a bigger subset of weird behaviour than just number literals.